package.json

```diff
@@ -45,7 +45,7 @@
   },
   "dependencies": {
     "@skyscanner/stylelint-plugin-backpack": "^1.1.0",
-    "prettier": "^1.19.1",
+    "eslint-config-skyscanner": "^8.0.0",
```

Comment:
Moved from `devDependencies` to `dependencies`, as we rely on `prettier` installed by `eslint-config-skyscanner`